django_project/monitor/serializers.py

```diff
@@ -190,3 +190,84 @@ def to_representation(self, instance):
         }
 
         return combined_data
+
+class ObservationsDataOnlySerializer(serializers.ModelSerializer):
+	"""Serializer of observation."""
+	collector_name = serializers.SerializerMethodField()
+	images = serializers.SerializerMethodField()
+	
+
+	class Meta:
+		model = Observations
```

Comment:
@tinashechiraya can you check the spacing, it looks different with the existing codes.